src/calculateRentalCost.js

```diff
@@ -4,7 +4,22 @@
  * @return {number}
  */
 function calculateRentalCost(days) {
-  // write code here
+  const RENT_COST = 40;
+  const SHORT_TERM = 3;
+  const LONG_TERM = 7;
+  const LONG_TERM_DISCOUNT = 50;
+  const SHORT_TERM_DISCOUNT = 20;
+  const totalRentalCost = days * RENT_COST;
```

Comment:
The variable `totalRentalCost` is appropriately declared as a `const` since its value is not being reassigned, which is a good practice.